src/main/kotlin/com/ampnet/projectservice/persistence/repository/ProjectRepository.kt

```diff
@@ -70,4 +70,6 @@ interface ProjectRepository : JpaRepository<Project, UUID> {
             "INNER JOIN project.organization organization WHERE organization.uuid = ?1"
     )
     fun countProjectsByOrganization(organizationUuid: UUID): Int
+
+    fun findAllByCoopAndActive(coop: String, active: Boolean = true): List<Project>
```

Comment:
This is not correct, `Project` can have the status `active` but the start or end date can be outside the current period so it won't be visible to the user. `active` is a flag that enables the project owner to show or hide the project.
Use: `findByActive`

Reply:
Hmm, did not notice that route. I wanted to check start/end date validity in the mail service, but It is better in here.